.gitignore

```diff
@@ -13,4 +13,5 @@ generated-src/
 .vscode/
 .project
 .settings
-.classpath
\ No newline at end of file
+.classpath
+.gitpod.yml
```

Comment:
What's this? o.O

Reply:
![image](https://user-images.githubusercontent.com/66781351/168113168-35500bbd-aa57-4116-8216-e82af57efcf8.png)

Reply:
Ok, I see.